react-native/ios/Podfile

```diff
@@ -0,0 +1,35 @@
+# Resolve react_native_pods.rb with node to allow for hoisting
+require Pod::Executable.execute_command('node', ['-p',
+  'require.resolve(
+    "react-native/scripts/react_native_pods.rb",
+    {paths: [process.argv[1]]},
+  )', __dir__]).strip
+
+platform :ios, min_ios_version_supported
+prepare_react_native_project!
+
+target 'RNSDKTester' do
+  config = use_native_modules!
+  pod 'ObjectiveDropboxOfficial', '6.2.3', :modular_headers => true
```

Comment:
Why is this here?

Reply:
I thought we use this dep.

Reply:
Yes but it's part of the RN SDK already. It doesn't need to be here, does it?

Reply:
I think it does:
![image](https://github.com/jitsi/jitsi-meet-sdk-samples/assets/26413496/9f2dc89d-7a78-4cc9-81ae-449ee125371a)

Reply:
Was that without it? Then we probably need to fix the podspec in the RN SDK I think.

Reply:
I think it's kind of the same situation with rn webrtc, isn;t it? We either add dropbox as a peer dep or we use it with modular_header => true in Podfile as it is.

Reply:
No because we o LG use it in the native side.